src/staticanalysis/bot/static_analysis_bot/report/phabricator.py

```diff
@@ -32,6 +35,9 @@ def __init__(self, configuration={}, *args, **kwargs):
         assert self.analyzers is not None, \
             'No analyzers setup on Phabricator reporter'
 
+        self.modes = configuration.get('modes', (MODE_COMMENT, ))
```

Comment:
We could make this singular, I guess there is no case where we want to publish both lint issues and comments